src/pages/VestingClaim.tsx

```diff
@@ -0,0 +1,191 @@
+import { Contract as EthersContract, ethers } from "ethers"
+import React, { ReactElement, useEffect, useState } from "react"
+import { SDL_TOKEN, SDL_TOKEN_ADDRESSES } from "../constants"
+import { Trans, useTranslation } from "react-i18next"
+import { commify, formatBNToString } from "../utils"
+import { notifyCustomError, notifyHandler } from "../utils/notifyHandler"
+import Button from "../components/Button"
+import INVESTOR_EMPLOYEE_VESTING_CONTRACT_ABI from "../constants/abis/vesting.json"
+import SDL_TOKEN_ABI from "../constants/abis/sdl.json"
+import { Sdl } from "../../types/ethers-contracts/Sdl"
+import TopMenu from "../components/TopMenu"
+import { Vesting } from "../../types/ethers-contracts/Vesting"
+import logo from "../assets/icons/logo.svg"
+import plusIcon from "../assets/icons/plus.svg"
+import styles from "../components/TokenClaimModal.module.scss"
+import { useActiveWeb3React } from "../hooks"
+import useAddTokenToMetamask from "../hooks/useAddTokenToMetamask"
+
+function VestingClaim(): ReactElement {
+  const { t } = useTranslation()
+  const { account, chainId, library } = useActiveWeb3React()
+  const { addToken, canAdd } = useAddTokenToMetamask({
+    ...SDL_TOKEN,
+    icon: `${window.location.origin}/logo.svg`,
+  })
+
+  const [isValidBene, setIsValidBene] = useState(false)
+  const [claimableVestedAmount, setClaimableVestedAmount] = useState("0.0")
```

Comment:
Prefer to keep raw `BigNumber`'s in state, and format them in UI. The reason is that we may need to do math or display multiple formats with those numbers, so we want to keep them in their pure form. Therefore your default value for both of these is `Zero` from `import {Zero} from "@ethersproject/constants"`